.github/workflows/CI.yaml

```diff
@@ -147,15 +166,48 @@ jobs:
 
       - name: "Test: Gear workspace"
         run: ./scripts/gear.sh test gear --release
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: JS metadata"
         run: ./scripts/gear.sh test js
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
+
+      - name: "Test: Process gear-test"
+        run: ./scripts/gear.sh test gtest
+        env:
+          RUSTFLAGS: -Cinstrument-coverage
 
       - name: "Test: Process node runtime testsuite"
         run: ./scripts/gear.sh test rtest
 
       - name: "Test: Runtime upgrade and queue processing"
         run: ./scripts/gear.sh test runtime-upgrade
+            
+      - name: "Coverage: Aggregate"
+        run: >-
+          ./grcov . --binary-path ./target/release/ -s . -t lcov --branch --ignore-not-existing
+          --ignore "/*"
```

Comment:
scripts/read.me-s/weight.rs/core-errors and dirs, which start with ".", should be also ignored

Reply:
> core-errors

Why? If unused variant appeared compiler won't say it to us because every enum is public but we will see it in coverage

Reply:
> scripts/read.me-s/weight.rs/core-errors and dirs, which start with ".", should be also ignored

you can see that only `*.rs` source files are included in coverage results, so only `weights.rs` and `migration.rs` should be ignored.